quickstarts/aws/aws-qldb/config.yml

```diff
@@ -15,14 +15,15 @@ description: |-
 
   ### More info
 
-  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-qldb-monitoring-integration) to learn more about New Relic monitoring for AWS QLDB. 
+  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-qldb-monitoring-integration) to learn more about New Relic monitoring for AWS QLDB.
```

Comment:
@bradleycamacho any suggestions on this description?

Reply:
I'd probably recommend including some details of how we describe this on the docs page

New Relic offers an integration for reporting your Amazon Quantum Ledger Database (QLDB) data to New Relic. This document explains how to activate this integration and describes the data that can be reported.